### ISSUE TITLE
[Suggestion] Fast tower scaffold for ncp

### ISSUE THREAD

**Issue by supakeks** [MeteorDevelopment/meteor-client#860]
**Describe the feature**
Make fast tower in scaffold work on ncp servers

**Comment by Cypphi** [comment]
I agree, maybe even a speed slider so we can go up like 20 bps on vanilla

**Comment by RedCarlos26** [comment]
It is possible to fast tower on ncp, mars client has it.

**Comment by BobIsMyManager** [comment]
i remeber mentioning this on the discord bruh
